chaos_genius/connectors/databricks.py

```diff
@@ -0,0 +1,95 @@
+"""Apache Databricks connector."""
+
+import pandas as pd
+from sqlalchemy import create_engine, text
+
+from chaos_genius.connectors.base_db import BaseDb
+
+from .connector_utils import merge_dataframe_chunks
+
+
+class Databricks(BaseDb):
+    """Apache Databricks connector."""
+
+    test_db_query = "SELECT 1"
+
+    def get_db_uri(self):
+        """Create SQLAlchemy URI from data source info."""
+        db_info = self.ds_info
+
+        if db_info is None:
+            raise Exception("Datasource info not found for Databricks.")
+
+        host = db_info.get("host")
+        access_token = db_info.get("access_token")
+        cluster_http_path = db_info.get("cluster_http_path")
+
+        if not (host and access_token and cluster_http_path):
+            raise Exception("Database Credential not found for Databricks.")
+
+        self.sqlalchemy_db_uri = f"databricks+connector://token:{access_token}@{host}:443"
+        return self.sqlalchemy_db_uri
+
+    def get_tables(self, use_schema=None):
```

Comment:
Any specific reason for overriding this method from BaseDB

Reply:
yeah u mean `get_tables` right, mentioned in comments (TODO) , default way errors out for schemas with no tables in Databricks somehow